core-web/apps/dotcms-ui/src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.spec.ts

```diff
@@ -77,6 +78,14 @@ describe('DotNavItemComponent', () => {
         expect(arrow.componentInstance.name).toBe('arrow_drop_up');
     });
 
+    it('should avoid label_important icon', () => {
+        componentHost.menu.tabIcon = LABEL_IMPORTANT_ICON;
+        fixtureHost.detectChanges();
+        const icon: DebugElement = de.query(By.css('dot-nav-icon'));
+
+        expect(icon.componentInstance.icon).not.toBe(LABEL_IMPORTANT_ICON);
+    });
```

Comment:
We are going to have the `LABEL_IMPORTANT_ICON` filtered out from the icon selector, because is a user select it will never see it and it will be a super weird reason.